sonic_platform_base/sonic_sfp/sfputilhelper.py

```diff
@@ -45,15 +45,15 @@ class SfpUtilHelper(object):
     def __init__(self):
         pass
 
-    def read_porttab_mappings(self, porttabfile, asic_inst=0):
+    def read_porttab_mappings(self, porttabfile, asic_inst=0, asic_name=None):
```

Comment:
Do we really need to add this additional parameter ?

Reply:
Currently, the existing parameter "asic_inst" use 0 as default for non-asic instance.  But 0 is is a valid instance ID for multiasic case.  And this function could be used in many place with the 0 as non-asic instance. We cannot simply enhance it for the mutliasic case.   Using an additional parameter is a simple solution to  identify the non-asic and multiasic case.

Reply:
Thinking again, @mlok-nokia I think we can avoid this parameter. Can we check if it is multi-asic and then pass asic_name to get_port_config API ?
``` 
ASIC_NAME_PREFIX = 'asic'

asic_name=none
if is_multi_asic:
    asic_name= ASIC_NAME_PREFIX++str(inst)
```

Reply:
Fixed as suggested above